Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -597,10 +597,10 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
             let pos = getMarkerPosition(highlight: highlight)
 
             // check bounds
-            if !_viewPortHandler.isInBounds(x: pos.x, y: pos.y)
-            {
-                continue
-            }
+//            if !_viewPortHandler.isInBounds(x: pos.x, y: pos.y)
```

Comment:
❔  Schvalne zakomentováno?

Reply:
Jojo, napíšu tam TODO. Chci to tam nechat. Potřebuji mít y na 0, abych mohl označit i část grafu mezi vrcholy. Bez y nastaveného na 0 jdou vybrat jen vrcholy.

Reply:
Nebo já to smažu, to bude lepší.